solr/docker/build.gradle

```diff
@@ -453,10 +453,10 @@ if (''.equals(releaseGpgFingerprint)) {
         exec {
           commandLine 'docker', 'run',
               '--cidfile', mockServerIdFile,
-              '--rm',
```

Comment:
why is `--rm` removed? This would leave containers around afterwards

Reply:
Good call, this was for debugging